make_tag.py

```diff
@@ -34,74 +34,26 @@ def format_version(major, minor, build):
 
 
 def get_next_version(bump_major, bump_minor):
-    # looking to parse tags of form: 'release-0.1.36-cutoff' to get the latest version number and increment build
-    # component.
-    # Because we have some inconsistency in the prefix (sometimes release-, sometimes release-venice-), assume tags are
-    # either of the two:
-    # 1. Full versions (e.g. '0.1.37')
-    # 2. End in '-<version>-cutoff' (e.g. '-0.1.37-cutoff')
-    tags_text = check_output(['git', 'tag'], text=True)  # tag1\ntag2\ntag3...
-    tags_lines = [str(l) for l in tags_text.splitlines()]  # ['tag1', 'tag2', ...]
-
-    full_version_regex = '(^\\d+(?:.\\d+)+$)'
-    legacy_version_regex = '^release-(\\d+(.\\d+)+)-cutoff$'
-    tags_numbers = []
-    for line in tags_lines:
-        full_version_regex_match = re.search(full_version_regex, line)
-        if full_version_regex_match:
-            tags_numbers.append(full_version_regex_match.group(1))
-            continue
-
-        legacy_version_regex_match = re.search(legacy_version_regex, line)
-        if legacy_version_regex_match:
-            tags_numbers.append(legacy_version_regex_match.group(1))
-            continue
-
-    version_numbers = [n.split('.') for n in tags_numbers if len(n.split('.')) == 3]  # [ ['0','1','36'], ...]
-    version_ints = [[int(part) for part in number] for number in version_numbers]  # [[0,1,36], ...]
-
-    max_major = 0
-    for v in version_ints:  # [0, 1, 36]
-        max_major = max(max_major, v[0])
-
-    if bump_major:
-        max_major += 1
-
-    major_filtered_version_numbers = [n for n in version_ints if n[0] == max_major]
+    tag_text = check_output(['git', 'describe', '--tags', '--abbrev=0', 'main'], text=True)
```

Comment:
We should check the tags on `remote` main. So, Let's use `check_output(['git', 'describe', '--tags', '--abbrev=0', f'{remote}/main'], text=True)`

Also, before this, we need a `git fetch <remote>` too

Reply:
sure, fetch is called before this.